dev-guide/cluster-fleet-evaluation.md

```diff
@@ -0,0 +1,160 @@
+that the change would have affected that cluster. You can join multiple
+_ConditionStatus/Reason_ to test multiple scenarios. Now release the detection
+logic to master and
+[backport](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit)
+it to a N-1 Z stream release where N is the current release under development.
+The Z stream backport is useful as it allows the team to gather the data without
+going through a full release cycle. The backport should follow the
+[OpenShift backport process](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit).
+How far back the backports should be done is subjective and the teams should aim
+to backport as far back as possible so as to collect a reasonable amount data
+that allows them to reach a conclusion.
+
+Once the Z streams have been released the component developers can [interrogate
+the telemetry data](#interacting-with-telemetry) to figure out the prevalence of
+the scenario that the change will introduce. Allow sufficient time (TBD) for
+customers to upgrade to the Z stream release. The upgrade data is also available
+in telemetry.
+
+At this point the team needs to validate that the telemetry data matches their
+expectations and make a decision if they still plan to move forward with the
+change. In particular the team needs to measure the level of impact the change
+has. If the percentage of impact is very low then it might be preferable to
+contact the affected customers. Alternatively if the percentage of impact is
+very high, it would be preferable that the team re-evaluate the change and
+investigate if there is a way to automate the fix process in a seamless manner.
+
+If the component team decides to move forward, they will then be responsible for
+writing a [Knowledge Centered Support](https://source.redhat.com/groups/public/cee-kcs-program/cee_kcs_program_wiki/kcs_solutions_content_standard_v20)
+(KCS) document providing sufficient information describing the scenario and how
+to resolve it. Refer to the [KCS section](#writing-a-kcs-document) for more
+information.
+
+Once the KCS article/solution is ready and the list of customers are available,
+the component team can alert the affected customers using the methods described
+in the [Alerting customers](#writing-a-kcs-document) reference section. Teams
+should make every attempt to use all methods of alerting customers (this ensures
+contact with customers is made), but are free to drop options if they feel using
+that method may not reach the proper audience or may be too expensive.
+
+**Note**: Before the team goes down this path it will be useful to discuss the
+outcome and expectations with their respective OpenShift staff engineer, product
+manager and PLM contact. PLM contacts are listed under *column S* in the
+[OCP Team Member Tracking sheet](https://docs.google.com/spreadsheets/d/1M4C41fX2J1nBXhqPdtwd8UP4RAx98NA4ByIUv-0Z0Ds/edit#gid=1382138347).
```

Comment:
and another internal doc link, which might benefit from `(internal link)` or some other marker